StortSpelProjekt/Engine/src/Renderer/D3D12Timer.cpp

```diff
@@ -134,19 +134,19 @@ namespace D3D12
 
 	void D3D12Timer::ResolveQueryToGPU(ID3D12GraphicsCommandList* pCommandList, ID3D12Resource** ppQueryResourceGPUOut)
 	{
-		setGPUResourceState(pCommandList, D3D12_RESOURCE_STATE_COPY_SOURCE, D3D12_RESOURCE_STATE_COPY_DEST);
+		setGPUMemorytate(pCommandList, D3D12_RESOURCE_STATE_COPY_SOURCE, D3D12_RESOURCE_STATE_COPY_DEST);
 
 		pCommandList->ResolveQueryData(m_pQueryHeap_, D3D12_QUERY_TYPE_TIMESTAMP, 0, m_TimerCount * 2, m_pQueryResourceGPU, 0);
 
-		setGPUResourceState(pCommandList, D3D12_RESOURCE_STATE_COPY_DEST, D3D12_RESOURCE_STATE_COPY_SOURCE);
+		setGPUMemorytate(pCommandList, D3D12_RESOURCE_STATE_COPY_DEST, D3D12_RESOURCE_STATE_COPY_SOURCE);
 
 		if (ppQueryResourceGPUOut)
 		{
 			*ppQueryResourceGPUOut = m_pQueryResourceGPU;
 		}
 	}
 
-	void D3D12Timer::setGPUResourceState(ID3D12GraphicsCommandList* pCommandList, D3D12_RESOURCE_STATES prevState, D3D12_RESOURCE_STATES newState)
+	void D3D12Timer::setGPUMemorytate(ID3D12GraphicsCommandList* pCommandList, D3D12_RESOURCE_STATES prevState, D3D12_RESOURCE_STATES newState)
```

Comment:
Missing "S" in name. Please explain why you did this name change

Reply:
This file was changed by misstake